app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java

```diff
@@ -263,16 +255,28 @@ public void bind(NavDrawerData.DrawerItem drawerItem) {
                         && feed.getImageUrl() != null && feed.getImageUrl().startsWith(Feed.PREFIX_GENERATIVE_COVER);
                 new CoverLoader(mainActivityRef.get())
                         .withUri(feed.getImageUrl())
-                        .withPlaceholderView(feedTitle, textAndImageCombind)
-                        .withCoverView(imageView)
+                        .withPlaceholderView(title, fallbackTitle, textAndImageCombind)
```

Comment:
Now I wonder why the main title field needs to be passed to the CoverLoader. Isn't it easier to just set the title here in SubscriptionsRecyclerAdapter (which you already do)?

Reply:
Title field is no longer passed.